viewer/packages/cad-styling/src/NodeIdNodeCollection.ts

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+import { CogniteClient, ListResponse, Node3D } from '@cognite/sdk';
+import chunk from 'lodash/chunk';
+import { CdfModelNodeCollectionDataProvider } from './CdfModelNodeCollectionDataProvider';
+import { CdfNodeCollectionBase } from './CdfNodeCollectionBase';
+import { NodeCollectionSerializationContext, registerNodeCollectionType } from './NodeCollectionDeserializer';
+import { SerializedNodeCollection } from './SerializedNodeCollection';
+
+/**
+ * Collection that holds a set of nodes including children identified by nodeIds. Note that
+ * this involves remapping IDs to "tree indices" and subtree sized used by Reveal using
+ * the CDF API. Its often better to use one of the other node collections or {@see TreeIndexNodeCollection}
+ * whenever possible for best performance.
+ */
+export class NodeIdNodeCollection extends CdfNodeCollectionBase {
```

Comment:
`NodeCollectionOfNodeId` could be better name?

Reply:
I tried to keep the style we have today - e.g. `TreeIndexNodeCollection`. Could probably consider all collections really...